packages/valory/skills/market_manager_abci/bets.py

```diff
@@ -261,6 +279,7 @@ def hook(data: Dict[str, Any]) -> Union[Bet, PredictionResponse, Dict[str, Bet]]
         # if this is a `Bet`
         bet_annotations = sorted(Bet.__annotations__.keys())
         if bet_annotations == data_attributes:
+            data["queue_status"] = QueueStatus(data["queue_status"])
```

Comment:
Let's ensure this is compatible with the old json version too, in case someone upgrades but already has some bets stored.

```suggestion
            if "queue_status" in data:
                data["queue_status"] = QueueStatus(data["queue_status"])
```